draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -671,151 +629,104 @@ This approach is the simplest solution, as it requires no interpretation by the
+This solution represents an advanced approach when a new opaque configuration attribute is mapped to multiple memory-map locations on a pluggable device, or when multiple such attributes are mapped to a single memory-map location on pluggable. Similar to {{solution-2}}, the mapping between these new attributes and their corresponding memory-map locations should be detailed in the pluggable Repository. For each new opaque attribute, the host platform is required to perform a lookup in the pluggable Repository to identify the relevant memory-map locations. The platform then assembles the corresponding values, which are communicated to the pluggable device via the CMIS protocol.
 
 Although this solution is included for completeness, it is not practical or desirable due to its complexity and the need for interpretation by the host software
 
-### Vendor-Specific Secret Capability Attributes
+### Support of Opaque Secret Configuration Attributes
 
-to be added
-
-### Vendor-Specific Secret Configuration Attributes
-
-There are situations where vendor-specific configuration attributes are confidential, and the vendor wishes to conceal their meanings and values. When considering the interface from the pluggable device to the host via CMIS, it is crucial that the pluggable does not expose the meaning or value of these confidential attributes. Ideally, the pluggable device would encrypt the data. Within the context of CMIS, it may be sufficient to allocate an array of register locations to convey the property values. These registers would store an encrypted data blob for read-only properties and accept an encrypted blob for writable registers. The specific value might be set or read through different register positions on each read/write, depending on the encryption technique used.
+There are situations where opaque configuration attributes are confidential, and the vendor wishes to conceal their meanings and values. When considering the interface from the pluggable device to the host via CMIS, it is crucial that the pluggable does not expose the meaning or value of these confidential attributes. Ideally, the pluggable device would encrypt the data. Within the context of CMIS, it may be sufficient to allocate an array of register locations to convey the property values. These registers would store an encrypted data blob for read-only properties and accept an encrypted blob for writable registers. The specific value might be set or read through different register positions on each read/write, depending on the encryption technique used.
```

Comment:
which vendors want to conceal and why are they confidential and to whom and for what purpose? I'd expect from a gap analysis a much clearer identification of the gap and why it needs to be defined in a standard.